src/admin/settings/settings.controller.js

```diff
@@ -0,0 +1,127 @@
+(function () {
+    'use strict';
+
+    angular.module('motech-admin')
+        .controller('SettingsController', settingsController);
+
+    settingsController.$inject = ['$scope', 'PlatformSettingsFactory', 'i18nService', '$http', 'ModalWindow', 'LoadingModal', 'ServerService'];
+    function settingsController ($scope, PlatformSettingsFactory, i18nService, $http, ModalWindow, LoadingModal, ServerService) {
+        $scope.comboboxValues = { "security.password.validator" : ["none", "lower_upper", "lower_upper_digit", "lower_upper_digit_special"],
+                                          "system.language" : ["en", "pl"] };
+        $scope.platformSettings = PlatformSettingsFactory.get();
+
+        $scope.label = function (key) {
+            return i18nService.getMessage('admin.settings.' + key);
+        };
+
+        $scope.saveSettings = function (settings) {
+            LoadingModal.open();
+            $http.post(ServerService.formatURL('module/admin/api/settings/platform'), settings)
+                .success( function() {
+                    LoadingModal.close();
+                    ModalWindow(angular.toJson(i18nService.getMessage('admin.settings.saved')),
```

Comment:
Is there a reason these calls to i18nService are wrapped in angular.toJson?

Since these are just success callbacks, it would make more sense to use [ModalAlert](https://github.com/motech/motech-ui/blob/master/src/common/modal/modal-alert.factory.js)